temporian/implementation/numpy/operators/tests/assign/BUILD

```diff
@@ -12,8 +12,11 @@ py_test(
     deps = [
         # absl/testing:absltest dep,
         "//temporian/implementation/numpy/operators:assign",
+        "//temporian/implementation/numpy/operators/tests/assign/test_data:assigned_repeated_timestamps",
```

Comment:
Nit: Make sure to fix formatting.

Reply:
Can you explain what should be fixed? I don't seem to realize. Is the order of the tests?

Reply:
(In github) I see the lines 15 and 16 as not aligned the same way as the other lines (e.g. 17-21).